### DISCUSSION TITLE
Auto follow a lead avatar in a room?

### DISCUSSION THREAD

**Discussion by captfarg** [mozilla/hubs#3444] [General Questions & Discussion] [answered]
Hey all!  I'd like to be able to have other people in a Hubs room automatically follow me as I tour around a scene.  

The use case is this:  

I have a Cathedral 3d model (realworld cathedral).  I want to give guided tours of this.  I want all avatars to automatically follow me as I move to different areas of the scene.  The problem right now is that inexperienced users have to learn an interface just to follow the 'guide' in a tour.  I'd like to remove this barrier by simply having them automatically follow after my avatar in the room.  That way, as long as they can find their way into a room, they can experience the guided tour without having to learn controls.    Possible?

**Comment by pattersonbl2** [comment, marked as answer]
I have opened a new issue for this discussion. https://github.com/mozilla/hubs/issues/4564. 
I will mark this as answered.
